README.md

```diff
@@ -65,7 +65,8 @@ docker run \
   -p 5432:5432 \
   -d tensorchord/vchord-postgres:pg17-v0.2.2
```

Comment:
```suggestion
  -d tensorchord/vchord-postgres:pg17-latest
```

Reply:
There is no latest tag. Will change to v0.3.0

Reply:
fixed